pkg/agent/storage/https.go

```diff
@@ -99,7 +99,8 @@ func (h *HTTPSDownloader) Download(client http.Client) error {
 		if err := extractZipFiles(resp.Body, fileDirectory); err != nil {
 			return err
 		}
-	} else if strings.Contains(contentType, "application/x-tar") || strings.Contains(contentType, "application/gzip") {
+	} else if strings.Contains(contentType, "application/x-tar") || strings.Contains(contentType, "application/x-gtar") ||
+		strings.Contains(contentType, "application/x-gzip") || strings.Contains(contentType, "application/gzip") {
```

Comment:
I think this condition can be simplified to 
```python
tar_content_types = ["application/x-tar", "application/x-gtar", "application/x-gzip"]
else if any(ct in contentType for ct in tar_content_types):
```

Reply:
I don't think Go supports such syntax...

Reply:
oops, sorry was thinking about storage initializer..